editor/src/viewport_tools/tools/shared/path_outline.rs

```diff
@@ -0,0 +1,115 @@
+use crate::document::DocumentMessageHandler;
+
+use graphene::layers::layer_info::LayerDataType;
+use graphene::layers::style::{self, Fill, Stroke};
+use graphene::{LayerId, Operation};
+
+use glam::DAffine2;
+use kurbo::{BezPath, Shape};
+use std::collections::VecDeque;
+
+use crate::consts::{COLOR_ACCENT, SELECTION_OUTLINE_WEIGHT};
+use crate::message_prelude::*;
```

Comment:
Our (arbitrary, but consistent) convention is to put the local (`crate::` and `super::`) imports first, followed then by a block for our first-party imports from other crates in the project, then followed by `std::` and third-party dependencies.